src/App.css

```diff
@@ -28,6 +28,34 @@
   color: #61dafb;
 }
 
+.title {
+  padding: 24px 24px 24px 24px;
+}
+
+.container {
+  display: flex;
+  height: 30px;
+  margin-bottom: 16px;
+  text-align: center;
+  padding: 0px 0px 0px 24px;
+}
+
+
+.id-container {
+  width:30px; 
+  border: 1px solid;
+  border-radius: 30px;
```

Comment:
I think the border-radius is only `10px` in the design

![image](https://user-images.githubusercontent.com/37349324/210093969-333cc5db-6d0f-4a4f-ac7c-b882bdc893b3.png)